vita3k/modules/SceHttp/SceHttp.cpp

```diff
@@ -320,9 +320,9 @@ EXPORT(SceInt, sceHttpCreateRequestWithURL, SceInt connId, SceInt method, const
     if (parsed.path.empty())
         parsed.path = slash;
     if (parsed.query.empty())
-        parsed.query = "";
+        parsed.query.clear();
```

Comment:
after some thought i think these are not needed anymore, i had them there because before it was a char pointer, but now its a string and it initializes it above so it starts as an empty string, then the url gets parsed onto the parsed object and then it is checking if the string is empty, if it is then clear it, it's like a redundant check imo. would be cool to also get rid of these i think

Reply:
How important is it to fix it in this PR?

Reply:
not really important, just some cleanup, you can skip it if you want